### PR TITLE
doc: PyTorch Lightning integration

### LINKED ISSUE #2882: docs: new page about dvclife integration with pytorch_lightning
I've made a [PR](https://github.com/iterative/dvclive/pull/168) with the feature. It's approved, so I write a documetation page about it.

### PR DESCRIPTION
**Pull request by sirily**
Fix #2882 

Documentation about the new [feature](https://github.com/iterative/dvclive/pull/168)

### CODE REVIEW

File: content/docs/dvclive/ml-frameworks/pytorch-lightning.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    path='my_logs_path',
+    summary=False
+)
+trainer = Trainer(
+    logger=dvclive_logger,
+)
+trainer.fit(model)
+```
+
+> 📖 PyTorch Lightning by default creates directory with the logger's name to
```
Comment (daavoo) [inline content/docs/dvclive/ml-frameworks/pytorch-lightning.md:65]: This reminded me that we should include logic for saving the model inside `DvcLiveLogger` to be consistent with the other integrations (which support the `model_file` arg).

**Review by daavoo — APPROVED** [review]

File: content/docs/dvclive/ml-frameworks/pytorch-lightning.md (hunk 1 above)
Comment (daavoo) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:65]: https://github.com/iterative/dvclive/issues/170

Comment (sirily) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md]: In Lightning saving is controlled by another callback and is independent of logger. I suppose, we can't include this logic and keep compatibility with native pytorch lightning checkpoint callbacks

Comment (sirily) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md]: `LightningModule` implements method `log`, which user call to store everything he wants, and then `LightningModule` calls logger. Checkpoints are saved at the end of batch/epoch via hooks like `on_train_batch_end`, etc. Logger doesn't has access to these hooks.
To save models we should write callback and encourage users to use it with our logger. But users can use native callbacks and control saving everything himself, and track it with DVC.

**Pull request merged**